gpu_bdb/bdb_tools/cluster_startup.py

```diff
@@ -35,8 +35,25 @@ def attach_to_cluster(config, create_sql_context=False):
     scheduler_file = config.get("scheduler_file_path")
     host = config.get("cluster_host")
     port = config.get("cluster_port", "8786")
+    start_local_cluster = config.get("start_local_cluster", False)
+
+    if start_local_cluster:
+        from dask_cuda import LocalCUDACluster
+        cluster = LocalCUDACluster(
+            n_workers=int(os.environ.get("NUM_WORKERS", 16)),
+            device_memory_limit=os.environ.get("DEVICE_MEMORY_LIMIT", "15GB"),
+            local_directory=os.environ.get("LOCAL_DIRECTORY"),
+            rmm_pool_size=os.environ.get("POOL_SIZE", "29GB"),
+            memory_limit=os.environ.get("DEVICE_MEMORY_LIMIT", "1546828M"),
+            enable_tcp_over_ucx=True,
+            enable_nvlink=os.environ.get("CLUSTER_MODE", "TCP")=="NVLINK",
+            protocol="ucx" if os.environ.get("CLUSTER_MODE", "TCP")=="NVLINK" else "tcp",
+            enable_infiniband=False,
+            enable_rdmacm=False
```

Comment:
We dont have `jit_unspill` enabled here.  
```suggestion
           enable_rdmacm=False,
           jit_unspill=True
```

Reply:
Read about https://docs.rapids.ai/api/dask-cuda/stable/spilling.html#jit-unspill hee.